h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -984,7 +984,11 @@ public Frame asFactor(Frame frame, String columnName) {
     return frame;
   }
 
-  public void printOutFrameAsTable(Frame fr, boolean rollups, long limit) {
+  public static void printOutFrameAsTable(Frame fr) {
```

Comment:
I think this is dangerous and should not be here, or it should set some reasonable default for limit like 100

Reply:
@honzasterba This is subjective. I can try to parse enormous file into Frame  with `TesUtil.parse_test_file`(and there are no reasonable defaults for this method AFAIN) and it will be tough for application/test to do it. It is on developer( as we are talking about test module) to decide what is dangerous and what is not. For me setting some 'reasonable` defaults ( magic numbers) is more dangerous as someone who will be using this method might not be aware of real size of the frame and might make wrong conclusions about correctness of what he is developing right now. To be safe then we will need to create bunch of methods like `printOutFirst100LinesFromFrame`, `printOutFirst200LinesFromFrame` etc. 
Main point here - it is in test package and it will not cause production issues by incidental use of this.